db_migrations.go

```diff
@@ -1993,4 +1993,42 @@ var migrations = []any{
 	newSqlMigration(`
         DROP TABLE network_client_connection_error
     `),
+
+	newSqlMigration(`
+		CREATE TABLE network_user_auth_sso
+		(
+			user_id uuid NOT NULL,
+			auth_type varchar(32) NOT NULL,
+			auth_jwt text NOT NULL,
+			user_auth varchar(256) NOT NULL,
+			create_time timestamp NOT NULL DEFAULT now(),
+
+			PRIMARY KEY (user_id, auth_type)
+		)
+	`),
+	newSqlMigration(`
+		CREATE TABLE network_user_auth_password
+		(
+			user_id uuid NOT NULL,
+			user_auth varchar(256) NOT NULL,
+			auth_type varchar(32) NOT NULL,
```

Comment:
Is `auth_type` in this table always set to `"password"`? I don't think we need it here, but it's ok to keep it.